src/heat-map/heat-map.component.ts

```diff
@@ -211,14 +218,14 @@ export class HeatMapComponent extends BaseChartComponent {
   }
 
   setColors(): void {
-    this.colors = new ColorHelper(this.scheme, 'linear', this.valueDomain);
+    this.colors = new ColorHelper(this.scheme, this.scaleType, this.valueDomain);
   }
 
   getLegendOptions() {
     return {
-      scaleType: 'linear',
+      scaleType: this.scaleType,
```

Comment:
I added the `scaleType` as an input to get ordinal color scales like:

![image](https://cloud.githubusercontent.com/assets/509946/24294973/17a771d6-10dc-11e7-8476-8e5e3b52f2e7.png)

I really didn't like the result, but thought it could be useful in other cases.  We can remove this option.